backend/dal/Services/Concrete/BuildingService.cs

```diff
@@ -44,14 +44,15 @@ public IEnumerable<Building> Get(double neLat, double neLong, double swLat, doub
             // Check if user has the ability to view sensitive properties.
             var userAgencies = this.User.GetAgencies();
             var viewSensitive = this.User.HasPermission(Permissions.SensitiveView);
+            var isAdmin = this.User.HasPermission(Permissions.AdminProperties);
 
             IQueryable<Building> query = null;
             // Users may only view sensitive properties if they have the `sensitive-view` claim and belong to the owning agency.
             query = this.Context.Buildings
                 .Include(b => b.Parcel)
                 .AsNoTracking()
                 .Where(p =>
-                (!p.IsSensitive || (viewSensitive && userAgencies.Contains(p.AgencyId))) &&
+                (isAdmin || !p.IsSensitive || (viewSensitive && userAgencies.Contains(p.AgencyId))) &&
```

Comment:
Doesn't have to happen in this PR but would love to see if you can find a way to extract the `WhoCanSee` functionality into an extension method or utils class or something to delegate this decision. 

Currently, we have to modify the code in multiple places (BuildingService, ParcelService, ParcelExtensions, etc etc) when we want to change/tweak who can see sensitive properties. 

The probability of bugs grows exponentially when we have to resort to copy/paste around the codebase for changes in logic.